app/server/src/main/scala/org/bitcoins/server/ServerJsonModels.scala

```diff
@@ -1403,7 +1404,35 @@ object OfferRemove {
         }
       case other =>
         val exn = new IllegalArgumentException(
-          s"Bad number or arguments to rejectincomingoffer, got=${other.length} expected=1")
+          s"Bad number or arguments to offer-remove, got=${other.length} expected=1")
+        Failure(exn)
+    }
+  }
+}
+
+case class OfferSend(
+    remoteAddress: InetSocketAddress,
+    message: String,
+    offerTLV: DLCOfferTLV)
```

Comment:
This needs to take a temporaryContractId so we can call it again if it doesn't go through the first time from the Contracts list where we don't have the DLCOfferTLV